src/plugins/data/public/search/expressions/__snapshots__/es_raw_response.test.ts.snap

```diff
@@ -0,0 +1,155 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`esRawResponse converts aggregations to table simple aggregation response 1`] = `
+Object {
+  "columns": Array [
+    Object {
+      "id": ".2.buckets.key",
```

Comment:
do we care about the preceding `.` at the beginning of the IDs here?

i'm sure it still works, but just wanted to confirm this was expected behavior for the `flatten` function

Reply:
fixed